config/BardinIMDRadCorPXSec.xml

```diff
@@ -5,8 +5,24 @@
   Configuration sets for the Bardin-Dokuchaeva inverse muon 
   decay differential xsec algorithm with 1-loop radiative corrections
 
+  <!--
+  Configuration for the BardinIMDRadCorPXSec xsec algorithm.
+
+  Configurable Parameters:
+  ....................................................................................
+  Name                    Type     Optional   Comment                         Default
+  XSec-Integrator         alg      No                                         genie::XSecOnElectron/Default 
+  Electron-Velocity       alg      No                                         genie::ElectronVelocityMap/Default
+  N-Integration-Samples   int      No                                         10000
+  NuE-XSecRelError        double   No                                         0.0001
```

Comment:
more description here. Explain here that N integration is the maximum number of iteration in case we don't reach the desired relative error